scripts/git-hooks/pre-commit

```diff
@@ -16,6 +16,9 @@ if [ -d ./venv/ ]; then
     fi
 fi
 
+echo "Validating branch name..."
+[[ $(git branch --show-current) =~ ^u/ ]] || EXIT=1
```

Comment:
This is a hard check right? So if I use the pre-commit hook (which honestly, I don't) then it will prevent me from committing to `prototype/1234-next-amazing-feature`? That seems less than ideal - maybe it should just print a warning message instead of aborting?